README.md

```diff
@@ -207,6 +207,41 @@ Please refer to [RFC 6749 §5.1 (Successful
 Response)](https://tools.ietf.org/html/rfc6749#section-5.1) for further details
 on the fields.
 
+#### Using a pop-up to drive the journey
+
+Clients may choose to drive the journey using a pop-up window. When loaded
+inside of a pop-up window, the following considerations apply to Fractal ID's
+flow.
+
+1. If the User is not logged in to Fractal ID, they'll see a login screen as
+   they normally would. After entering their email address, the pop-up window
+will close itself in 5 seconds.
+1. If the User is logged in to Fractal ID, they'll be presented with the normal
+   journey _inside the pop-up_ window.
```

Comment:
Add: `, if they still haven't filled all required data; otherwise, they will simply see the authorization screen`